packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -0,0 +1,88 @@
+import * as cdk from '@aws-cdk/core';
+import { CfnPortfolio } from './servicecatalog.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from 'constructs';
+
+/**
+ * A Service Catalog portfolio.
+ */
+export interface IPortfolio extends cdk.IResource {
```

Comment:
I deliberately omitted the `portfolioName` attribute. From what I can tell, this is just the display name of the Portfolio, and it allows us to have a `fromPortfolioArn()` method, instead of a `fromPortfolioAttributes()` one where you have to specify both the ARN and the name yourself.

Is there any API that requires passing the name of the Portfolio? Can we skip it?

Reply:
We need the `portfolioName`, you wouldn't be able to deploy the stack without it.

Reply:
Can you expand - where exactly is it used?